### DISCUSSION TITLE
Webstudio - open source framer/webflow alternative

### DISCUSSION THREAD

**Discussion by x0rb** [coollabsio/coolify#4656] [Service Template Requests]
There's not much alternative for a powerful, self-hosted website builder that's comparable to framer and webflow.

Here's the doc https://docs.webstudio.is/university/self-hosting

**Comment by bst1n** [comment]
Now this is ready :

https://github.com/webstudio-is/webstudio/pull/4744

**Comment by simplerethink** [comment]
Is there a Coolify service template for Webstudio yet? I see an [integration how-to in the docs](https://coolify.io/docs/integrations/webstudio), but if I'm understanding it correctly, it assumes you're using the cloud version of Webstudio.
